frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -227,148 +224,43 @@ class TagEditorParamInner extends Component<Props> {
     const hasSelectedDimensionField =
       isDimension && Array.isArray(tag.dimension);
     const hasWidgetOptions = widgetOptions?.length > 0;
```

Comment:
```suggestion
    const hasWidgetOptions = widgetOptions.length > 0;
```

Reply:
Ok, agreed, but this code I haven't even touched 😅